packages/components/psammead-image/package-lock.json

```diff
@@ -46,7 +46,7 @@
       "dev": true,
       "requires": {
         "jest-styled-components": "^6.3.1",
-        "react-test-renderer": "^16.8.6"
+        "react-test-renderer": "^16.6.3"
```

Comment:
Was this intended? I don't understand why this is getting downgraded here

Reply:
Lol no idea, fixed :)